src/modules/acbs/acbs-facility-loan-transaction.service.get-loan-transaction-by-bundle-identifier.test.ts

```diff
@@ -0,0 +1,155 @@
+import { HttpService } from '@nestjs/axios';
+import { PROPERTIES } from '@ukef/constants';
+import { GetFacilityLoanTransactionGenerator } from '@ukef-test/support/generator/get-facility-loan-transaction-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { AxiosError } from 'axios';
+import { when } from 'jest-when';
+import { of, throwError } from 'rxjs';
+
+import { DateStringTransformations } from '../date/date-string.transformations';
+import { AcbsFacilityLoanTransactionService } from './acbs-facility-loan-transaction.service';
+import { AcbsException } from './exception/acbs.exception';
+import { AcbsResourceNotFoundException } from './exception/acbs-resource-not-found.exception';
+
+describe('AcbsFacilityLoanTransactionService', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const dateStringTransformations = new DateStringTransformations();
+  const idToken = valueGenerator.string();
+  const baseUrl = valueGenerator.httpsUrl();
+
+  let httpService: HttpService;
+  let service: AcbsFacilityLoanTransactionService;
+
+  let httpServiceGet: jest.Mock;
+
+  beforeEach(() => {
+    httpService = new HttpService();
+
+    httpServiceGet = jest.fn();
+    httpService.get = httpServiceGet;
+
+    service = new AcbsFacilityLoanTransactionService({ baseUrl }, httpService);
+  });
+
+  describe('getLoanTransactionByBundleIdentifier', () => {
+    const portfolioIdentifier = PROPERTIES.GLOBAL.portfolioIdentifier;
```

Comment:
we can destructure this as well

Reply:
I decided to do a find `const portfolioIdentifier = PROPERTIES.GLOBAL.portfolioIdentifier;` replace `const { portfolioIdentifier } = PROPERTIES.GLOBAL;` to change this everywhere in the codebase.